ensenso_camera_msgs/action/CalibrateHandEye.action

```diff
@@ -35,6 +35,21 @@ uint8 command
 # calibration patterns.
 string parameter_set
 
+# Optional for the step START_CALIBRATION:
+# The underlying robot geometry. By default a 6-DOF robot is assumed. In case
+# you are calibrating a 4- or 3-DOF robot, some of the axes have to be fixed.
+# With a 4-DOF robot you can either fix the camera pose Z-component or the
```

Comment:
Finde ich gut mit der Liste von Geometrien, das ist nicht so schwierig zu benutzen, wie die API in der NxLib.

Allerdings könnte es jetzt noch irgendwelche 4 DOF Roboter geben, die andere Achsen nicht haben als die x/y Rotation. Das klappt nur für die genaue Situation, die in https://www.ensenso.com/manual/3.2/commands/calibratehandeye.html beschrieben ist. Andere Situationen bräuchten dann neue Modi. Das sollten wir evtl. noch klarstellen.